setup.cfg

```diff
@@ -24,7 +24,7 @@ project_urls =
 packages = find:
 install_requires =
     apache-airflow>=1.10.0
-    firebolt-sdk>=1.0.0a
+    firebolt-sdk>=1.1.0
```

Comment:
Should we require here a version with httpx version fix?

Reply:
I don't think we should require it - it's only needed for some of the MWAA versions. Pip would resolve to higher versions in case pydantic does not work with 1.1.0. Airflow can also be deployed outside of AWS, where pydantic/httpx might not be pinned.
Here I limited it to the lowest acceptable version - the one containing v1 and v2 merge so the features are supported.